satpy/readers/yaml_reader.py

```diff
@@ -823,19 +823,31 @@ def _load_dataset_with_area(self, dsid, coords, **kwargs):
         if not file_handlers:
             return
 
-        area = self._load_dataset_area(dsid, file_handlers, coords, **kwargs)
-
         try:
```

Comment:
LGTM, but how will moving this below affect reading other netcdf data? We rely on the tests?

Reply:
we do rely on tests yes, and these are failing. I have to take a look at the problem